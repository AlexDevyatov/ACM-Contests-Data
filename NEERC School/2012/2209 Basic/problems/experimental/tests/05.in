9 9
1 9
9 2
4 3
9 7
6 4
4 7
5 4
7 6
4 8
1 1 0 2 0 2 2 1 0
