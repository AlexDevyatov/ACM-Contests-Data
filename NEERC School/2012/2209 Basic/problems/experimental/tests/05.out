1 2 4 7 6 7 4 6 8
